### PR TITLE
Point API to dev instance of FireCloud

### PR DESCRIPTION
**Pull request by dmohs**
Tested by creating a new account, signing-in, then creating a workspace.

### CODE REVIEW

File: api/tools/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ def workbench_db_password = System.getenv("WORKBENCH_DB_PASSWORD")
 
 def dbProperties = [
   'spring.datasource.driver-class-name': 'com.mysql.jdbc.Driver',
-  'spring.datasource.url': 'jdbc:mysql://${db_host}:${db_port}/workbench',
+  'spring.datasource.url': 'jdbc:mysql://${db_host}:${db_port}/workbench?useSSL=false',
```
Comment (danqrodney) [inline api/tools/build.gradle]: What is this change for?

Comment (dmohs) [reply api/tools/build.gradle]: Reduces log noise and makes this consistent with the other one.

Comment (dmohs) [reply api/tools/build.gradle]: Also, our database isn't configured to allow outside connections (which is good), so our connections are either local or through the proxy, which encrypts all traffic.

Comment (dmohs) [reply api/tools/build.gradle]: Based on your question, I decided this was worth a comment.

**Pull request merged**